pkgs/development/libraries/nuspell/default.nix

```diff
@@ -1,17 +1,17 @@
-{ stdenv, fetchFromGitHub, cmake, pkgconfig, boost, icu, catch2, ronn }:
+{ stdenv, fetchFromGitHub, cmake, pkgconfig, boost, icu, catch2, ronn, pandoc }:
```

Comment:
If they switched to pandoc, `ronn` is probably not necessary anymore? Their readme also doesn't state ronn as a dependency.

Reply:
You're right. It seems to build fine without it. I pushed the change.